scripts/README.md

```diff
@@ -0,0 +1,65 @@
+# Protego Scripts
+
+## Usage
+
+### 1. Install dependencies
+
+```
+npm i
+```
+
+### 2. Run scripts
+
+List the 20 most recent plans in `MCD_PAUSE`
```

Comment:
I believe that instead of fixing an arbitrary number of events to return, we could maybe let the user specify how many blocks they want to go in the past.

However, that introduces one edge case: events for the same  `[usr, tag, fax, eta]` might be missing because of the `from` block. It's usually fine, as we can consider any subsequent `exec` or `drop` to have been preceded by a `plan`.

Reply:
Done in [f6e02c3](https://github.com/makerdao/protego/pull/5/commits/f6e02c35c395a800babd075a2fee46d2c24a0bf8)